kubecost.yaml

```diff
@@ -20317,28 +20303,6 @@ subjects:
     name: kubecost-cost-analyzer
     namespace: kubecost
 ---
-# Source: cost-analyzer/templates/cost-analyzer-psp-rolebinding.template.yaml
-apiVersion: rbac.authorization.k8s.io/v1
-kind: RoleBinding
-metadata:
-    name: kubecost-cost-analyzer-psp
```

Comment:
Wouldn't we also need to remove the PSP itself, not just the role and rolebinding that used them?

Reply:
good find! I'm looking at this now.